.github/workflows/test_e2e.yaml

```diff
@@ -9,22 +9,32 @@ on:
       - "v*" # Push events to matching v*, i.e. v1.0, v20.15.10
     branches:
       - main
+  schedule:
+    - cron: "0 8 * * *"
```

Comment:
So that means we're going to trigger these e2e tests on main every morning then? If there a specific even that triggered this change, or we're just being extra careful?

Reply:
> we're just being extra careful

On scheduled run the matrix is wider (all python, all runners (macos added)). But yeah, it's more or less being extra careful